redis/jsonresource_test.go

```diff
@@ -0,0 +1,69 @@
+package redis
+
+import (
+	"context"
+	"testing"
+
+	"github.com/alicebob/miniredis/v2"
+	"github.com/datatrails/go-datatrails-common/logger"
+	"github.com/go-redis/redis/v8"
+	"github.com/stretchr/testify/require"
+)
+
+// NewTestResource sets up a fresh instance of miniredis and returns a configured JsonResource
+func NewTestResource(t *testing.T) JsonResource {
+	mr := miniredis.RunT(t)
+	c := redis.NewClient(&redis.Options{Addr: mr.Addr()})
+	return JsonResource{
```

Comment:
are there any mutexes in underlying structs for JsonResource?
if so return a pointer instead of a copy...